schemas/2015-08-01/Microsoft.Compute.json

```diff
@@ -1592,6 +1596,212 @@
         "settings",
         "protectedSettings"
       ]
+    },
+    "VMSnapshot":{
```

Comment:
Synced offline. "location" and "tags" are defined in base types in "deploymentTemplate.json". Resource types can override them if needed.